readme.MD

```diff
@@ -10,16 +10,29 @@ A simple CLI tool for backporting commits
 
 
 ### Install
+
 ```
 npm install -g backport
 ```
 
 ### Usage
+
 ```
 > backport
 ```
 
+### Options
+
+| Option | Description | Accepts |
+| --- | --- | --- |
+| --multiple     | Backport to multiple versions |         [boolean] [default: true] |
+| --own          | Only show own commits |                 [boolean] [default: true] |
+| --config       | Show configuration file |                               [boolean] |
+| --help         | Show help |                                             [boolean] |
+| -v, --version  | Show version number |                                   [boolean] |
```

Comment:
Awesome! I don't think we need to write `[boolean]` for `config`, `help` and `version`. Apart from that 👍